autockt/envs/ngspice_ledro_d_fc45.py

```
@@ -0,0 +1,496 @@
+from collections import OrderedDict
+import yaml
+import yaml.constructor
+import statistics
+import os
+import IPython
+import itertools
+from eval_engines.util.core import *
+import pickle
+import os
+
+from eval_engines.ngspice.TwoStageClass import *
```

Ruff (0.12.2)

`from eval_engines.ngspice.TwoStageClass import *` used; unable to detect undefined names

(F403)